### DISCUSSION TITLE
[Idea] csv 文档生成助手或者插件

### DISCUSSION THREAD

**Discussion by scortLi** [lobehub/lobehub#1074] [Ideas]
### 🥰 需求描述

我需要获取数据然后将数据保存在类似于scv等文档中，包括本地或者线上（需要可下载），有类似的助手或者插件支持吗

### 🧐 解决方案

我需要获取数据然后将数据保存在类似于scv等文档中，包括本地或者线上（需要可下载），有类似的助手或者插件支持吗

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
目前应该没有

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
话说 scv 是啥？

**Reply by scortLi** [reply]
不好意思，打错了，是csv 文件，就是普通的数据保存在文本中，openai的assistant就支持的
